### ISSUE TITLE
Disable saving of series image to each chapter file?

### ISSUE THREAD

**Issue by murraypaul** [Deathspike/mangarack#54]
One thing I find odd is that mangarack doesn't actually save the chapter as it is actually shown on the site (at least for Batato). It adds an extra page at the front of each chapter, using the overall series image. This is frequently the cover image for (one of) the most recent chapters or volumes updated, which means that every chapter downloaded at the same time has the same cover image, and it is wrong for almost all of them.
It also means that the cover shown for any given chapter file depends on when you download it, as it will change whenever the series cover image gets updated.
This doesn't seem to fit the desire for behaviour to be consistent and predictable, and I'd rather have no image than the wrong one.

**Comment by Deathspike** [comment]
This is a sore point. I completely and utterly agree with everything you said. That said, without a nice cover, the series listing in ComicRack will look like utter crap. I can deal with the fact that the chapter listing previews wouldn't be very good, but at least series stacks should be quite obvious as to what the series actually is.

If there is a way to convince ComicRack of a series cover w/o embedding it in each chapter.. I'd be very happy with that.

**Comment by XellossMetallium** [comment]
For this maybe I have a suggestion, but obviously I don't know if it is feasible.

This site here: http://mcd.iosphe.re/ should provide a a vast, open and stable archive of manga covers (actually also back covers, sides etc.). 
The search goes by mangaupdates series id.
Since I don't think there is a way to desume MU ID from batoto (or there is?) it has to be manually added, maybe near the link (just like the filters before, but this wouldn't be a filter).
So something like:
http://bato.to/comic/_/comics/tegami-bachi-r93 --mu15690
http://bato.to/comic/_/comics/tegami-bachi-r93 --MCD(15690)

at this point the behavior should be:
if volume number is provided download cover by volume, 
if not take the first (or last?) volume cover, 
if no MU ID is set go as always (from batoto).

I suppose this would be quite nice and accurate (at the cost of a bit of manual work, I suppose).. 
The site should have been created to work for this purpose, so implementation should be easy.. but I say nothing more, since as you know it is not my field.

**Comment by murraypaul** [comment]
Personally I think that functionality belong in a ComicRack pluging for people who want it to run after the fact, rather than modifying the downloaded files to include something extra.

**Comment by XellossMetallium** [comment]
Yes, and actually simply putting the aforementioned ID in metadata would work too.. the problem, I think, is that most occidental comics are distribuited with cover included. Once I tried to ask for support for mangaupdates info scraper (I came to understand that such plugins are relatively easy to make) but my suggestion didn't get support at all (and this is how I started using Mangarack). 
Userbase for manga seems to be very small (probably because of the great number of online readers), so any work in such direction will have to come from here.. be it as a feature in Mangarack or as a plugin in Comicrack.. 
In other words, being practical, more than expelling this feature from MR I think it is better to keep it under control, improve it (if possible) and to add a switch to deactivate it, should someone prefer to keep the file unaltered or to modify manually just the first chapter to keep the looks of the series stack.
